src/transformers/models/bert/modeling_flax_bert.py

```diff
@@ -669,4 +661,43 @@ def __call__(
             rngs=rngs,
         )
 
-        return logits, pooled
+        return (logits,)
+
+
+class FlaxBertForMaskedLMModule(nn.Module):
+    vocab_size: int
+    hidden_size: int
+    intermediate_size: int
+    head_size: int
+    num_heads: int
+    num_encoder_layers: int
+    type_vocab_size: int
+    max_length: int
+    dropout_rate: float = 0.0
+    dtype: jnp.dtype = jnp.float32
+
+    @nn.compact
+    def __call__(
+        self, input_ids, attention_mask=None, token_type_ids=None, position_ids=None, deterministic: bool = True
+    ):
+        # Model
+        encoder = FlaxBertModule(
+            vocab_size=self.vocab_size,
+            type_vocab_size=self.type_vocab_size,
+            hidden_size=self.hidden_size,
+            intermediate_size=self.intermediate_size,
+            head_size=self.hidden_size,
+            num_heads=self.num_heads,
+            num_encoder_layers=self.num_encoder_layers,
+            max_length=self.max_length,
+            dropout_rate=self.dropout_rate,
+            dtype=self.dtype,
+            add_pooling_layer=False,
+            name="bert",
+        )(input_ids, attention_mask, token_type_ids, position_ids, deterministic=deterministic)
+
+        # Compute the prediction scores
+        encoder = nn.Dropout(rate=self.dropout_rate)(encoder, deterministic=deterministic)
+        logits = FlaxBertOnlyMLMHead(vocab_size=self.vocab_size, name="cls", dtype=self.dtype)(encoder)
+
+        return logits
```

Comment:
Will make `run_mlm_flax.py` in the examples crash I think, the script is looking for both `logits` and `pooled` outputs

Reply:
Yeah, however `pooled` is never used by the `ForMaskedLM` models, it should only be used for the `ForPreTraining` models. In `run_mlm_flax.py` pooled was expected as the output, but never actually used => so I just removed it there. 

FlaxBertForMaskedLM should not output a `pooled` tensor, cf. with https://github.com/huggingface/transformers/blob/3552d0e0d89711404ec49b0080cd1f48ae224bb3/src/transformers/models/bert/modeling_bert.py#L1200